tests/unit/types/arrays/test_documentarray.py

```diff
@@ -332,6 +332,29 @@ def test_da_sort_by_document_interface_in_proto():
     assert da[0].embedding.shape == (1,)
 
 
+def test_da_sort_topk():
```

Comment:
what is the behavior when top-k is present but no key is given?

Reply:
Both with and without `top_k`, if `key=None` n exception is raised
```
TypeError: '<' not supported between instances of 'DocumentProto' and 'DocumentProto'
```

Reply:
That was my fear. I think then we should not consider `key` as an Optional `argument`

Reply:
Sure I can make it required. We 'll break the API but I think it raises the same Exception in `master` so we 're good, correct?

Reply:
yes, we are good because `key` would be accepted as a `*kwargs` so good